### ISSUE TITLE
Can we please rename RpcClient module 

### ISSUE THREAD

**Issue by wanglongfei88** [neo-project/neo-modules#175]
RpcClient module is actually a light SDK which could be solely referenced for dApp development. But it is called RpcClient and be included in neo-modules. So the naming is little bit confusing for users or developers who want to use it as a neo plugin. So I will recommend we give it a more suitable and meaningful name. For example, RpcSDK or RpcClientSDK, etc.

**Comment by erikzhang** [comment]
> But it is called RpcClient and be included in neo-modules.

That's why we rename the repo from `neo-plugins` to `neo-modules`. I think it is not confusing.

**Comment by wanglongfei88** [comment]
Hi, @erikzhang , I understand we rename the repo to `neo-modules`, but 'RpcClient' is not very clear for developers to identify what's usage of it that is why some developers asked us what is it RpcClient and what is the usage.  

I think `RpcSDK` or `RpcClientSDK` maybe more clear and easier for developers to know what it is, and hopefully more developers then will use it.

**Comment by vncoelho** [comment]
I agree with @wanglongfei88, current names are not good: RpcServer and RpcClient.
What are the functionalities expected to each of then, Lonfgei?
In addition, we need to thing how it will looks like when we have Rest, for example.

**Comment by superboyiii** [comment]
Totally agree with @wanglongfei88 , it's important to let developers know what's that then they is able to use it. Otherwise, even it's a perfect project but people don't know it, then they'll not use it.
